poker.js

```diff
@@ -45,17 +45,21 @@ const groupedHand = (hand) => hand.reduce(
 const straight = (hand) => {
   const onlyNumbers = lettersToNumber(hand);
   const sortRankArr = onlyNumbers.sort((a, b) => a - b);
-  const lastMinusFirst = sortRankArr[4] - sortRankArr[0];
-  return lastMinusFirst;
-};
-
-const highCard = (rankArr) => {
-  let highestCard = 0;
-  for (let elem of rankArr) {
-    highestCard < elem ? highestCard = Number(elem) : highestCard;
-  }
-  return highestCard;
-};
+  const truthComparison = sortRankArr.reduce(
+    (acc, cv) => {
```

Comment:
Make arguments understandable as well as the function name. When another developer (or you after six months) the code must present its purpose and not looking for the context of its use. It's ok if the variable names are big. The purpose is to help the humans who work with your code.